app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
The code does not validate that the input string contains exactly three elements separated by spaces and that the first element is 'cp'. This is required by the checklist to ensure correct input format.